skbio/stats/ordination/__init__.py

```diff
@@ -113,12 +113,13 @@
 from ._correspondence_analysis import CA
 from ._redundancy_analysis import RDA
 from ._canonical_correspondence_analysis import CCA
-from ._principal_coordinate_analysis import PCoA
+from ._principal_coordinate_analysis import PCoA, PCoABase
```

Comment:
If this is only necessary for benchmarking, can we just remove this import just before the PR is merged?
The benchmarking will be complete by then correct?